exercises/etl/runtests.jl

```diff
@@ -0,0 +1,36 @@
+using Base.Test
+
+include("etl.jl")
+
+@testset "a single letter" begin
+    input = Dict(1=>["A"])
+    output = Dict("a"=>1)
+    @test transform(input) == output
+end
+
+@testset "single score with multiple letters" begin
+    input = Dict(1=>["A", "E", "I", "O", "U"])
+    output = Dict("a"=>1, "e"=>1, "i"=>1, "o"=>1, "u"=>1)
+    @test transform(input) == output
+end
+
+@testset "multiple scores with multiple letters" begin
+    input = Dict(1=>["A", "E"], 2=>["D", "G"])
+    output = Dict("g"=>2, "e"=>1, "a"=>1, "d"=>2)
+    @test transform(input) == output
+end
+
+@testset "multiple scores with differing numbers of letters" begin
+    input = Dict(1=>[ "A", "E", "I", "O", "U", "L", "N", "R", "S", "T" ],
```

Comment:
There are only single letters as key dictionaries, so they should be Chars 'A' etc. instead of Strings